packages/graphql/tests/integration/aggregations/top-level/alias.int.test.ts

```diff
@@ -67,24 +67,119 @@ describe("aggregations-top_level-alias", () => {
+                                    _longest: longest
+                                }
+                                _imdbRating: imdbRating {
+                                    _min: min
+                                    _max: max
+                                    _average: average
+                                }
+                                _createdAt: createdAt {
+                                    _min: min
+                                    _max: max
+                                }
+                            }
                         }
-                        _title: title {
-                            _shortest: shortest
-                            _longest: longest
-                        }
-                        _imdbRating: imdbRating {
-                            _min: min
-                            _max: max
-                            _average: average
+                    }
+                }
+            `;
+
+        const gqlResult = await testHelper.executeGraphQL(query);
+
+        expect(gqlResult.errors).toBeUndefined();
+
+        expect(gqlResult.data).toEqual({
+            [typeMovie.operations.connection]: {
+                aggr: {
+                    n: {
+                        _count: 4,
+                        _id: {
+                            _shortest: "1",
+                            _longest: "4444",
+                        },
+                        _title: {
+                            _shortest: "1",
+                            _longest: "4444",
+                        },
+                        _imdbRating: {
+                            _min: 1,
+                            _max: 4,
+                            _average: 2.5,
+                        },
+                        _createdAt: {
+                            _min: minDate.toISOString(),
+                            _max: maxDate.toISOString(),
+                        },
+                    },
+                },
+            },
+        });
+    });
+
+    test("using multiple aliased aggregate projections", async () => {
+        const typeDefs = `
+            type ${typeMovie} @node {
+                testString: ID!
+                title: String!
+                imdbRating: Int!
+            }
+        `;
+
+        const testString = generate({
+            charset: "alphabetic",
+            readable: true,
+        });
+
+        const minDate = new Date();
+
+        const maxDate = new Date();
+        maxDate.setDate(maxDate.getDate() + 1);
+
+        await testHelper.initNeo4jGraphQL({ typeDefs });
+
+        await testHelper.executeCypher(
+            `
+                    CREATE (:${typeMovie} {testString: "${testString}", id: "1", title: "1", imdbRating: 1, createdAt: datetime("${minDate.toISOString()}")})
+                    CREATE (:${typeMovie} {testString: "${testString}", id: "22", title: "22", imdbRating: 2, createdAt: datetime()})
```

Comment:
Even if these tests were already made, in my opinion, it will cover more cases if some entries are filtered out by the `testString` filter!

Reply:
As you said, those cases are already covered. And these tests are based on the existing ones, Improving tests on aggregations is a bit out of scope for this, already quite big, PR. But I'll keep this in mind as we do need to improve these tests once the aggregations work is complete